File: Problem001/go/main.go
package main

import (
	"fmt"
)

func main() {
	sum := 0
	for i := 1; i < 1000; i++ {
		if i%3 == 0 {
			sum += i
			continue
		}
		if i%5 == 0 {
			sum += i
			continue
		}
	}
	fmt.Println(sum)
}

File: Problem001/csharp/main.cs
using System;

class mainapp{
	public static void Main() {
		int sum = 0;
		for(int i = 1; i < 1000; i++){
			if(i%3 == 0 || i%5 == 0){
				sum += i;
			}
		}
		Console.WriteLine(sum);
	}
}